### ISSUE TITLE
Problem with sphinx-issues

### ISSUE THREAD

**Issue by adosar** [readthedocs/readthedocs.org#11314]
## Details

* Read the Docs project URL: https://readthedocs.org/projects/moxel
* Build URL (if applicable): https://readthedocs.org/projects/moxel/builds/24305594 /
* Read the Docs username (if applicable): https://readthedocs.org/profiles/adosar/

## Expected Result

I am able to build the docs without problem locally by running `make html`.

* `.readthedocs.yaml`
```yaml
# Read the Docs configuration file for Sphinx projects
# See https://docs.readthedocs.io/en/stable/config-file/v2.html for details

# Required
version: 2

# Set the OS, Python version and other tools you might need
build:
  os: ubuntu-22.04
  tools:
    python: "3.12"
    # You can also specify other tool versions:
    # nodejs: "20"
    # rust: "1.70"
    # golang: "1.20"

# Build documentation in the "docs/" directory with Sphinx
sphinx:
  configuration: docs/source/conf.py
  # You can configure Sphinx to use a different builder, for instance use the dirhtml builder for simpler URLs
  # builder: "dirhtml"
  # Fail on all warnings to avoid broken references
  # fail_on_warning: true

# Optionally build your docs in additional formats such as PDF and ePub
# formats:
#    - pdf
#    - epub

# Optional but recommended, declare the Python requirements required
# to build your documentation
# See https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html
python:
   install:
   - requirements: docs/requirements.txt
```

* `docs/requirements.txt`
```
pymatgen
matplotlib
pyvista
tqdm
fire
numpy
numpydoc
Sphinx
sphinx-argparse
sphinx-code-tabs
sphinx-copybutton
sphinx-rtd-theme
sphinxcontrib-applehelp
sphinxcontrib-devhelp
sphinxcontrib-htmlhelp
sphinxcontrib-jquery
sphinxcontrib-jsmath
sphinxcontrib-qthelp
sphinxcontrib-serializinghtml
sphinxemoji
sphinx-issues
```
* `docs/source/conf.py`
```python
# Configuration file for the Sphinx documentation builder.
#
# For the full list of built-in configuration values, see the documentation:
# https://www.sphinx-doc.org/en/master/usage/configuration.html

# -- Project information -----------------------------------------------------
# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information

import sys, os
import subprocess

sys.path.insert(0, os.path.abspath('../../src'))

project = 'MOXελ'
copyright = '2023-2024, Antonios P. Sarikas'
author = 'Antonios P. Sarikas'
release = f'{subprocess.run(["git", "describe"])}'

# -- General configuration ---------------------------------------------------
# https://www.sphinx-doc.org/en/master/usage/configuration.html#general-configuration

extensions = [
        'sphinx.ext.autodoc',
        'sphinx.ext.viewcode',
        'sphinx.ext.napoleon',
        'sphinxemoji.sphinxemoji',
        'sphinx_copybutton',
        'sphinxarg.ext',
        'sphinx_code_tabs',
        'sphinx_issues',
        ]

templates_path = ['_templates']
exclude_patterns = ['modules.rst']

# Exclude input prompts from copybutton
copybutton_exclude = '.linenos, .gp, .go'

# -- Options for HTML output -------------------------------------------------
# https://www.sphinx-doc.org/en/master/usage/configuration.html#options-for-html-output

html_theme = 'sphinx_rtd_theme'
#html_static_path = ['_static']

# Path to GitHub repo {group}/{project}  (note that `group` is the GitHub user or organization)
issues_github_path = "adosar/moxel"

# which is the equivalent to:
issues_uri = "https://github.com/{group}/{project}/issues/{issue}"
issues_prefix = "#"
issues_pr_uri = "https://github.com/{group}/{project}/pull/{pr}"
issues_pr_prefix = "#"
issues_commit_uri = "https://github.com/{group}/{project}/commit/{commit}"
issues_commit_prefix = "@"
issues_user_uri = "https://github.com/{user}"
issues_user_prefix = "@"
```

You can view the whole project tree [here](https://github.com/adosar/moxel).

## Actual Result
This is the error I get. Maybe it is related to [this](https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1069257#8).
```python
Running Sphinx v7.3.7
0.1.1

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/moxel/envs/0.1.1/lib/python3.12/site-packages/sphinx/registry.py", line 453, in load_extension
    mod = import_module(extname)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/docs/.asdf/installs/python/3.12.0/lib/python3.12/importlib/__init__.py", line 90, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen importlib._bootstrap>", line 1381, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1354, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1318, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'sphinx_issues'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/home/docs/checkouts/readthedocs.org/user_builds/moxel/envs/0.1.1/lib/python3.12/site-packages/sphinx/cmd/build.py", line 332, in build_main
    app = Sphinx(args.sourcedir, args.confdir, args.outputdir,
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/docs/checkouts/readthedocs.org/user_builds/moxel/envs/0.1.1/lib/python3.12/site-packages/sphinx/application.py", line 229, in __init__
    self.setup_extension(extension)
  File "/home/docs/checkouts/readthedocs.org/user_builds/moxel/envs/0.1.1/lib/python3.12/site-packages/sphinx/application.py", line 402, in setup_extension
    self.registry.load_extension(self, extname)
  File "/home/docs/checkouts/readthedocs.org/user_builds/moxel/envs/0.1.1/lib/python3.12/site-packages/sphinx/registry.py", line 456, in load_extension
    raise ExtensionError(__('Could not import extension %s') % extname,
sphinx.errors.ExtensionError: Could not import extension sphinx_issues (exception: No module named 'sphinx_issues')

Extension error:
Could not import extension sphinx_issues (exception: No module named 'sphinx_issues')
```

**Comment by rffontenelle** [comment]
Looking at the [raw build log](https://readthedocs.org/api/v2/build/24305594.txt), sphinx_issues is not getting installed by pip. This build run is related to commit [2cdc4b](https://github.com/adosar/moxel/commit/2cdc4b), which does not have 'sphinx-issues' listed in the [docs/requirements.txt](https://github.com/adosar/moxel/blob/2cdc4bbd0fe627ef00b3783b85d937e658da1a44/docs/requirements.txt) file.

**Comment by rffontenelle** [comment]
I see your "Fix docs" commit in master branch ([0a954cf](https://github.com/adosar/moxel/commit/0a954cfd7201c2bc4abb4bce5119dd2d25cbf2a2)) adds sphinx-issues, while the "Fix docs" in 0.1.1 tag does not ([2cdc4b](https://github.com/adosar/moxel/commit/2cdc4b)).
